### DISCUSSION TITLE
Can't sign up on a GitLab instance

### DISCUSSION THREAD

**Discussion by RaptaG** [tutao/tutanota#6019] [Bugs]
I get the following error:

![1 error prohibited this user from being saved: Email is not allowed for sign-up. Please use your regular email address. Check with your administrator.](https://github.com/tutao/tutanota/assets/77157639/26ba9c1e-7222-4bae-8a88-2822bd0320a5)

**Comment by charlag** [comment]
Thanks for letting us know but you should check in with people who admin that instance, it's not something we have influence over
